packages/models.py

```diff
@@ -0,0 +1,94 @@
+from django.db                  import models
+
+# Create your models here.
+class Package(models.Model):
+    name           = models.CharField(max_length=100)
+    description    = models.CharField(max_length=2000)
+    price          = models.CharField(max_length=50)
+    thumnail_image = models.CharField(max_length=2000)
+    sales_volume   = models.IntegerField()
+    category       = models.ForeignKey('Category',on_delete=models.CASCADE)
+    product        = models.ManyToManyField('Product',related_name='packages')
+    tag            = models.ManyToManyField('Tag',related_name='packages')
+    class Meta:
+        db_table = 'packages'
+
+class Product(models.Model):
+    name        = models.CharField(max_length=50)
+    description = models.CharField(max_length=2000)
+    brand_name  = models.CharField(max_length=200)
+    image_url   = models.CharField(max_length=2000)
+    kcal        = models.IntegerField()
+    nutrition   = models.CharField(max_length=500)
+
+    class Meta:
+        db_table = 'products'
+
+class Category(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = 'categories'
+
+class Package_image(models.Model):
+    image_url = models.CharField(max_length=2000)
+    packages  = models.ForeignKey('Package')
+
+    class Meta:
+        db_table = 'package_images'
```

Comment:
class명은 camelCase를 사용해 주세요!

```suggestion
class PackageImage(models.Model):
    image_url = models.CharField(max_length=2000)
    packages  = models.ForeignKey('Package')

    class Meta:
        db_table = 'package_images'
```

Reply:
수정했습니다.